R/file_analysis.R

```diff
@@ -0,0 +1,59 @@
+source("R/connect.R")
+source("R/utils.R")
```

Comment:
This is more a "knowledge of R" thing for me but: Is it strictly necessary to source files that are adjacent to each other like this?

I never seemed to needed to when working on the RStudio addin, but there, the docs also seemed to claim that you're not supposed to - but maybe that's limited to the RStudio addin stuff

Reply:
Well, I wrote my first line of R code like three weeks ago, so I'm not sure either. But I know for sure that if I start the R REPL in the projects root directory and then use, e.g., `load_all(".")`, it does not work if I don't specify a _root-relative_ path in `source`.

Reply:
I would tend to not use `source`, using a local install version of the package, or individual sources for it to work. I am not aware of a big package doing that + we may just use [roxygen2](https://roxygen2.r-lib.org/reference/tags-namespace.html) to help us out.

Reply:
Resolved in 8f2796d